src/components/header/TopNavBar.jsx

```diff
@@ -81,7 +98,11 @@ class TopNavBar extends Component {
 
         {/*Searchbar, bookmarkbtn, cartbtn*/}
         <Pane>
-          <SearchInput placeholder="SEARCH" width="160px" className="searchbar"/>
+          <SearchInput
+            placeholder="SEARCH"
+            width="160px"
```

Comment:
why also here?

Reply:
same answer as previous and evergreen components support inline stylings